src/validators.ts

```diff
@@ -76,22 +114,37 @@ class ValidatorPkg {
     for (const filepath of dbFiles) {
       shell.rm("-f", filepath);
     }
+
+    // reset the Validator config file in case it was modified with a custom
+    // Registry hardhat port
+    if (ORIGINAL_VALIDATOR_CONFIG) {
```

Comment:
the original file ends with a new line, but the file gets overwritten with no new line char at the end. thus, running LT will show a file has changed, even though it really hasn't. two things come to mind:
- add a `\n` to the end of the file
- use an eol override for eslint and remove it from prettier (i dont _think_ it has a specific file/eol rule i can use)